schedule/yast/minimal+base/minimal+base@yast-s390x.yaml

```diff
@@ -27,4 +25,8 @@ schedule:
   - installation/logs_from_installation_system
   - installation/reboot_after_installation
   - boot/reconnect_mgmt_console
+  - console/system_prepare
```

Comment:
I guess this cannot work, we need this module to be scheduled after `first_boot`